pkg/vpcmodel/explainabilityConnectivity.go

```diff
@@ -29,9 +29,30 @@ type rulesConnection struct {
 	egressRules  rulesInLayers
 }
 
-// loadBalancerDennyEgressRule - is a rule applied to all private IPs of a given load balancer:
+// the load balancer is a rule applied to all private IPs of a given load balancer:
 // these private IPs can only connect to pool members of the load balancer
-type loadBalancerDenyEgressRule bool
+type LoadBalancerRule struct {
+	// the relevant load balancer:
+	lb LoadBalancer
+	//	Denny- true if src is pip, and dst is not pool member:
+	denny bool
```

Comment:
Why not have such a rule only when it is relevant - deny is true? Why this additional bool property?